autogpt_platform/backend/backend/blocks/io.py

```diff
@@ -56,6 +53,12 @@ class Input(BlockSchema):
             advanced=True,
         )
 
+        def generate_schema(self):
+            schema = self.get_field_schema("value")
+            if possible_values := self.placeholder_values:
+                schema["enum"] = possible_values
+            return schema
```

Comment:
This looks like it belongs on `AgentDropdownInputBlock`?

Reply:
`placeholder_values` is part of the generic AgentInputBlock too